analyzers/tests/SonarAnalyzer.UnitTest/Helpers/SonarLintXmlReaderTest.cs

```diff
@@ -90,10 +101,25 @@ public void SonarLintXmlReader_PartiallyMissingProperties_ExpectedAndDefaultValu
     public void SonarLintXmlReader_CheckEmpty_DefaultBehaviour() =>
         CheckSonarLintXmlReaderDefaultValues(SonarLintXmlReader.Empty);
 
+    [TestMethod]
+    public void SonarLintXmlReader_LanguageDoesNotExist_Throws()
+    {
+        var sut = CreateSonarLintXmlReader($"ResourceTests\\SonarLintXml\\All_Properties_cs\\SonarLint.xml");
+        sut.Invoking(x => x.IgnoreHeaderComments("fsharp"))
+            .Should()
+            .Throw<UnexpectedLanguageException>()
+            .WithMessage("Language 'fsharp' is not supported.");
+
+        sut.Invoking(x => x.AnalyzeGeneratedCode("fsharp"))
+            .Should()
+            .Throw<UnexpectedLanguageException>()
+            .WithMessage("Language 'fsharp' is not supported.");
```

Comment:
This can fin on a single line, and then the emty line above also will not be needed